flixel/FlxGame.hx

```diff
@@ -758,8 +759,6 @@ class FlxGame extends Sprite
 		}
 		FlxArrayUtil.clearArray(FlxG.swipes);
 		#end
-		
-		filters = filtersEnabled ? _filters : null;
```

Comment:
Maybe, but again: that's a breaking change with the potential for stuff to silently stop working. Really worth it here? Doubt it...